sentinel-core/src/main/java/com/alibaba/csp/sentinel/config/SentinelConfig.java

```diff
@@ -42,37 +42,33 @@
     public static final int APP_TYPE_COMMON = 0;
 
     private static final Map<String, String> props = new ConcurrentHashMap<>();
+
     private static int appType = APP_TYPE_COMMON;
+    private static String appName = "";
 
-    public static final String APP_TYPE = "csp.sentinel.app.type";
+    public static final String PROJECT_NAME_PROP_KEY = "project.name";
+    public static final String APP_NAME_PROP_KEY = "csp.sentinel.app.name";
+    public static final String APP_TYPE_PROP_KEY = "csp.sentinel.app.type";
     public static final String CHARSET = "csp.sentinel.charset";
     public static final String SINGLE_METRIC_FILE_SIZE = "csp.sentinel.metric.file.single.size";
     public static final String TOTAL_METRIC_FILE_COUNT = "csp.sentinel.metric.file.total.count";
     public static final String COLD_FACTOR = "csp.sentinel.flow.cold.factor";
     public static final String STATISTIC_MAX_RT = "csp.sentinel.statistic.max.rt";
     public static final String SPI_CLASSLOADER = "csp.sentinel.spi.classloader";
 
-    static final String DEFAULT_CHARSET = "UTF-8";
-    static final long DEFAULT_SINGLE_METRIC_FILE_SIZE = 1024 * 1024 * 50;
-    static final int DEFAULT_TOTAL_METRIC_FILE_COUNT = 6;
-    static final int DEFAULT_COLD_FACTOR = 3;
-
-    private static String appName = "";
-    public static final String APP_NAME = "project.name";
-    public static final String SUN_JAVA_COMMAND = "sun.java.command";
-    private static final String JAR_SUFFIX_LOWER = ".jar";
-    private static final String JAR_SUFFIX_UPPER = ".JAR";
-
-    public static final int DEFAULT_STATISTIC_MAX_RT = 4900;
+    public static final String DEFAULT_CHARSET = "UTF-8";
+    public static final long DEFAULT_SINGLE_METRIC_FILE_SIZE = 1024 * 1024 * 50;
+    public static final int DEFAULT_TOTAL_METRIC_FILE_COUNT = 6;
+    public static final int DEFAULT_COLD_FACTOR = 3;
+    public static final int DEFAULT_STATISTIC_MAX_RT = 5000;
```

Comment:
Hhh, so now the max RT is 5000, and not the classic value 4900?

Reply:
The magic 4900 often makes users confusing, so I've changed it to 5000. The attribute might be deprecated in future versions.